src/atoms/inputs/select.stories.tsx

```diff
@@ -95,8 +98,10 @@ export default {
 
 const TemplateSingle: ComponentStory<typeof Select<'single'>> = (args) => {
   const [value, setValue] = useState<SelectOption>();
+  const ref = useRef<any>();
+  console.debug('ref', ref);
```

Comment:
@ArinCantCode ref is always undefined here, I know we already discussed it, but would appreciate you taking a closer look here

Reply:
what do you mean? From what i recall ref is always undefined on first render and gets set later. 🤔 or is it something else?

Reply:
@ArinCantCode  so that is correct and something I overlooked, however this remains undefined even after selecting/unselecting options etc.